sg_otio/sg_cut_track_writer.py

```diff
@@ -1121,3 +1123,48 @@ def _create_input_media_version(self, input_media, track_name, sg_project, sg_li
             "sg_uploaded_movie"
         )
         return version, published_file
+
+    @staticmethod
+    def get_local_storage_relative_path(local_storage, file_path):
+        """
+        Given a Local Storage and a file path, returns the relative path
+        to the file from the Local Storage, if any.
+
+        :param file_path: Absolute and normalized path to a directory or a file
```

Comment:
Does the path really needs to be normalized? It seems to handle that in the method?